pulp_container/tests/functional/conftest.py

```diff
@@ -0,0 +1,67 @@
+from urllib.parse import urljoin
+
+import pytest
+import requests
+
+from pulp_container.tests.functional.utils import (
+    TOKEN_AUTH_DISABLED,
+    AuthenticationHeaderQueries,
+    BearerTokenAuth,
+)
+
+
+@pytest.fixture
+def fetch_response(bindings_cfg):
+    """Return a response considering token authentication."""
+
+    def _fetch_response(method, url, **kwargs):
```

Comment:
It is in a separate file because `fetch_response` is used in two different places (conftest.py, test_push_signatures) and we are importing some classes from `pulp_container.tests.functional.utils`.

Reply:
I would expect `local_registry` to be used for "fundamental" podman operations, not for working with raw requests.

Reply:
Well all of the things in the other conftest are also only used for the functional tests, right? Maybe that is the right location for the conftest after all.
Isn't that special token handling only interesting to interactions with our local_registry? But you may be right in that we do not need podman installed to do this, but we need it to use the local_registry fixture.
My concern with this fixture is, that the name does not speak to me. It should reflect that we talk to the registry and that we handle the authentication. And ideally it would get the relative path only and add the local registry automatically.